include/reactor-uc/platform/pico/uart_channel.h

```diff
@@ -0,0 +1,39 @@
+#ifndef REACTOR_UC_PICO_UART_CHANNEL_H
+#define REACTOR_UC_PICO_UART_CHANNEL_H
+
+#include "reactor-uc/network_channel/uart_channel.h"
+#include "reactor-uc/network_channel.h"
+#include "reactor-uc/environment.h"
+
+#include "pico/stdlib.h"
+#include "hardware/uart.h"
+#include "hardware/irq.h"
+
+typedef struct FederatedConnectionBundle FederatedConnectionBundle;
+typedef struct UartPolledChannel UartPolledChannel;
+typedef struct UartAsyncChannel UartAsyncChannel;
+
+#define UART_CHANNEL_BUFFERSIZE 1024
+// The UartChannel is not connection-oriented and will always appear as connected, so no need to wait.
+#define UART_CHANNEL_EXPECTED_CONNECT_DURATION SEC(5)
+
+extern UartPolledChannel *uart_channel_0;
```

Comment:
Why are these here in the header file? Are they used anywhere outside `uart_channel.c`